lib/Doctrine/Common/EventManager.php

```diff
@@ -68,7 +70,7 @@ public function dispatchEvent($eventName, EventArgs $eventArgs = null)
      *
      * @return array The event listeners for the specified event, or all event listeners.
      */
-    public function getListeners($event = null)
+    public function getListeners(?string $event = null): array
```

Comment:
The difference is that for one, the scalar type hint says "I want a string", while for the other it says "I want a string or null". The "feature" where adding a default type of `null` makes anything nullable should not overrule a concise scalar type declaration, which in this case is `?string`.